src/client-apps/nuxeo-drive/nuxeo-drive-metrics-documentation.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Nuxeo Drive metrics
+description: Nuxeo Drive custom metrics documentation.
+tree_item_index: 900
+review:
+  comment: 'release'
+  date: '2021-03-30'
+  status: ok
+toc: true
+---
+
+In the Nuxeo Drive [release 5.1.0](https://github.com/nuxeo/nuxeo-drive/blob/master/docs/changes/5.1.0.md) usage metrics have been added using custom HTTP headers. Those metrics are sent to the Nuxeo platform, exclusively when the application calls specific operations or endpoints.
+
+On this page, you will find the details about these metrics and their content.
+
+Some of these metrics are marked with the `ASYNC` tag, meaning that they are sent in a specific process to avoid spamming the platform with too many requests. More documentation about this can be found in the [Nuxeo Drive asynchronous metrics](#Nuxeo-Drive-asynchronous-metrics) section.
```

Comment:
```suggestion
Some of these metrics are marked with the `ASYNC` tag, meaning that they are sent in a specific process to avoid spamming the platform with too many requests. More documentation about this can be found in the [asynchronous metrics](#asynchronous-metrics) section.
```